rsocket/statemachine/RSocketStateMachine.cpp

```diff
@@ -461,28 +461,27 @@ void RSocketStateMachine::handleConnectionFrame(
       }
 
       if (coldResumeInProgress_) {
-        auto streamIds = resumeManager_->getRequesterRequestStreamIds();
-        for (auto streamId : streamIds) {
-          auto consumerAllowance =
-              resumeManager_->getConsumerAllowance(streamId);
-          auto streamToken = resumeManager_->getStreamToken(streamId);
-          auto subscriber = coldResumeHandler_->handleRequesterResumeStream(
-              streamToken, consumerAllowance);
-          // TODO(somatsun): ensure that subscription is called from the correct
-          // thread (eventBase) without using EventBaseManager
-          streamsFactory().createStreamRequester(
-              yarpl::make_ref<ScheduledSubscriptionSubscriber<Payload>>(
-                  std::move(subscriber),
-                  *folly::EventBaseManager::get()->getEventBase()),
-              streamId,
-              consumerAllowance);
-        }
-
         streamsFactory().setNextStreamId(
             resumeManager_->getLargestUsedStreamId());
-
+        for (auto it : resumeManager_->getStreamResumeInfos()) {
```

Comment:
You want `auto&`, `it` here is a `std::pair<StreamId, StreamResumeInfo>` which you don't want to copy.